### PR TITLE
82 support for other plans premium as default

### LINKED ISSUE #82: Other plans
**GIVEN** a logged in User
**WHEN** I access the url `/subscriptions/new`
**THEN** I should see the premium plan information

---

**GIVEN** a logged in User
**WHEN** I access the url `/subscriptions/new?plan=premiummob`
**THEN** I should see 
```
AgileVentures Premium Mob Plan
```

---

**GIVEN** a logged in User
**WHEN** I access the url `/subscriptions/new?plan=premium_f2f`
**THEN** I should see 
```
AgileVentures Premium Face 2 Face Plan
```

### PR DESCRIPTION
**Pull request by mattwr18**
#### Issue addressed

fixes #82

### CODE REVIEW

**Comment by mattwr18** [comment]
@joaopapereira if you merge https://github.com/AgileVentures/WebsiteOne-FE/pull/71 there will be minimal difference between this and `develop`

File: src/components/Subscriptions.js [hunk 1]
```diff
@@ -7,6 +7,17 @@ import createBillingAgreement from '../helpers/createBillingAgreement'
 import queryString from 'query-string'
 import '../assets/Subscriptions.css'
 
+let membership = props => {
+  let info
+  if (queryString.parse(props.location.search).plan === 'premiummob') {
```
Comment (Kachulio1) [inline src/components/Subscriptions.js]: you can have a variable on line 11, `const plan = queryString.parse(props.location.search).plan'

File: src/components/Subscriptions.js [hunk 2]
```diff
@@ -17,17 +28,16 @@ export const Subscriptions = props => {
     }
   })
 
-  const { plan } = queryString.parse(props.location.search)
   return (
     <Fragment>
       <Container>
         <Header as='h1'>
-          AgileVentures {plan.charAt(0).toUpperCase() + plan.slice(1)}{' '}
+          AgileVentures {membership(props).name}{' '}
```
Comment (Kachulio1) [inline src/components/Subscriptions.js]: You could do the same here coz you're using name more than one's

File: src/components/Subscriptions.js [hunk 3]
```diff
@@ -7,7 +7,21 @@ import createBillingAgreement from '../helpers/createBillingAgreement'
 import queryString from 'query-string'
 import '../assets/Subscriptions.css'
 
+let membership = props => {
```
Comment (joaopapereira) [inline src/components/Subscriptions.js:10]: This is clearly something that should come from the backend. But we will do that in a different story

**Review by joaopapereira — APPROVED** [review]
I am ok with this change

**Pull request merged**